cmd/kubelet/app/options/options.go

```diff
@@ -76,6 +76,9 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.TLSPrivateKeyFile, "tls-private-key-file", s.TLSPrivateKeyFile, "File containing x509 private key matching --tls-cert-file.")
 	fs.StringVar(&s.CertDirectory, "cert-dir", s.CertDirectory, "The directory where the TLS certs are located (by default /var/run/kubernetes). "+
 		"If --tls-cert-file and --tls-private-key-file are provided, this flag will be ignored.")
+	fs.StringVar(&s.BootstrapKubeconfig, "bootstrap-kubeconfig", s.BootstrapKubeconfig, "Path to a kubeconfig file that will be used to get client certificate for kubelet. "+
```

Comment:
can we prefix this flag with experimental or alpha like we do with other experimental flags?